### DISCUSSION TITLE
%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/PL/%E7%BC%96%E8%AF%91%E8%AF%AD%E8%A8%80%E8%AE%BE%E8%AE%A1%E5%92%8C%E7%A8%8B%E5%BA%8F%E5%88%86%E6%9E%90/BrownCSCI1260/

### DISCUSSION THREAD

**Discussion by giscus** [MingLLuo/CS-Plan#11] [Announcements]
# %E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/PL/%E7%BC%96%E8%AF%91%E8%AF%AD%E8%A8%80%E8%AE%BE%E8%AE%A1%E5%92%8C%E7%A8%8B%E5%BA%8F%E5%88%86%E6%9E%90/BrownCSCI1260/

https://cs-plan.com/%E5%9F%BA%E7%A1%80%E6%B7%B1%E5%85%A5/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/PL/%E7%BC%96%E8%AF%91%E8%AF%AD%E8%A8%80%E8%AE%BE%E8%AE%A1%E5%92%8C%E7%A8%8B%E5%BA%8F%E5%88%86%E6%9E%90/BrownCSCI1260/

**Comment by rodrigosImposter** [comment]
请问录屏在哪呢？找了很多地方
